examples/log_test.py

```diff
@@ -1,5 +1,4 @@
 from pyspecdata import *
-import logging
```

Comment:
because I have it returning the logger object, which has info, debug, etc, methods, there is no longer a need to import logging.

this time, do feel free to resolve my comments as you read them, as long as you know what's going on, and post a comment once you've had a chance to review everything, and you can verify that this works for you.

Reply:
makes sense

Reply:
really, this works, but I'm trying to make sure it's worthwhile in terms of what you get out of it, so once everything is clear to you for a conversation, go ahead and resolve.  I will wait to merge until you resolve

Reply:
this code is not working for me. will you take a look at my last merge and make sure everything is up to snuff? I don't understand why it's not working... error message is in previous comment

Reply:
the problem was that you mixed up `level`, which is what I called it and `stringLevel`, which is what you had called it